graphene_sqlalchemy/fields.py

```diff
@@ -129,20 +129,31 @@ def get_query(cls, model, info, sort=None, **args):
         return query
 
 
-class BatchSQLAlchemyConnectionField(UnsortedSQLAlchemyConnectionField):
+class BatchSQLAlchemyConnectionField(SQLAlchemyConnectionField):
```

Comment:
I'm not sure what our exact plan in regards to unifying these fields was. I just made this sortable, we can discuss this again

Reply:
Looks good.
If I'm correct, this would remove the need for the separate unsorted field as sorting for all the possible fields could be disabled using the `Meta` options. Maybe there is a need to add a QoL `sort=None` shortcut for the  options.

Reply:
I deprecated the UnsortedSQLAlchemyConnectionField as you can just set `sort=None` on the meta class already to disable sorting: https://github.com/graphql-python/graphene-sqlalchemy/blob/6417061998823fb3531def8fee394bd81c6f4c24/graphene_sqlalchemy/tests/test_sort_enums.py#L263